views/components/SignUpForm.js

```diff
@@ -1,39 +1,59 @@
 import React from 'react';
+import AlphaSignUpForm from './AlphaSignUpForm';
+import BetaSignUpForm from './BetaSignUpForm';
 import FormField from './FormField';
 
 const SignUpForm = props => {
-  const { header, subhead, partnerId } = props;
+  const { header, subhead, partnerId, hideAlpha, showBeta } = props;
```

Comment:
could we maybe have the verbs consistent for hide/show? like going with both `showAlpha` and `showBeta`?